src/main/resources/i18n/Resources_nl.properties

```diff
@@ -66,22 +53,19 @@ supportPaneReportIssueSentence=Problemen? Suggesties?
 supportPaneReportIssueLinkSentence=Creëer ticket hier.
 uploadPaneStopButtonText=Stop
 uploadPaneUploadMoreButtonText=Upload meer...
-#TODO translate
-directoryStructureSupplierProgressTitle=Looking for files
+directoryStructureSupplierProgressTitle=Zoeken naar bestanden
 cloudAlbumsProviderProgressTitle=Het laden van albums in Google Photos
-#TODO translate
-uploaderAlbumProgressTitle=Adding to albums
+uploaderAlbumProgressTitle=Toevoegen aan albums
 uploaderFileProgressTitle=Media bestanden aan het uploaden.
-#TODO translate
-albumManagerProgressStatusTitle=Reconciling albums with Google Photos
+albumManagerProgressStatusTitle=Albums vergelijken met Google Foto's
```

Comment:
Google Photos is the name of the service, and is left in English everywhere else. Is it best not to translate it? If not, could you translate it everywhere please?

Reply:
true